### ISSUE TITLE
[Feature] List of failed tests with folded logs

### ISSUE THREAD

**Issue by levsa** [test-summary/action#27]
Is it possible to generate a folded list of failed tests, clickable to open an see the logs? This would give a better overview of the test failures, with possibility to open and see logs if needed.

**Comment by braingram** [comment]
I didn't see this issue until now. Is https://github.com/test-summary/action/pull/53 what you're hoping for?

**Comment by levsa** [comment]
> I didn't see this issue until now. Is #53 what you're hoping for?

Yes! An overview of failed tests with expandable tracebacks is really helpful when find and fixing test failures! 

Thanks!
